src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -314,6 +320,458 @@ resource trigger_exportContainer 'Microsoft.DataFactory/factories/triggers@2018-
+              type: 'WebActivity'
+              dependsOn: [
+                {
+                  activity: 'Get Token'
+                  dependencyConditions: [
+                    'Succeeded'
+                  ]
+                }
+                {
+                  activity: 'Set Open Month Export Name'
+                  dependencyConditions: [
+                    'Succeeded'
+                  ]
+                }
+              ]
+              policy: {
+                timeout: '0.12:00:00'
+                retry: 0
+                retryIntervalInSeconds: 30
+                secureOutput: false
+                secureInput: false
+              }
+              userProperties: []
+              typeProperties: {
+                url: {
+                  value: 'https://@{variables(\'resourceManagementUri\')}@{item().scope}/providers/Microsoft.CostManagement/exports/@{variables(\'exportName\')}?api-version=2021-10-01'
+                  type: 'Expression'
+                }
+                method: 'PUT'
+                headers: {
+                  authorization: {
+                    value: 'Bearer @{activity(\'Get Token\').output.access_token}'
+                    type: 'Expression'
+                  }
+                }
+                body: {
+                  value: '{\n  "properties": {\n    "schedule": {\n      "status": "Active",\n      "recurrence": "Daily",\n      "recurrencePeriod": {\n        "from": "@{utcNow()}",\n        "to": "2099-12-31T00:00:00Z"\n      }\n    },\n    "partitionData": "True",\n    "format": "Csv",\n    "deliveryInfo": {\n      "destination": {\n        "resourceId": "@{pipeline().parameters.StorageAccountId}",\n        "container": "msexports",\n        "rootFolderPath": "@{item().scope}"\n      }\n    },\n    "definition": {\n      "type": "amortizedcost",\n      "timeframe": "BillingMonthToDate",\n      "dataSet": {\n        "granularity": "Daily"\n      }\n    }\n  }\n}'
+                  type: 'Expression'
+                }
+              }
+            }
+            {
+              name: 'Set Open Month Export Name'
+              type: 'SetVariable'
+              dependsOn: [
+                {
+                  activity: 'If Azure Commercial'
+                  dependencyConditions: [
+                    'Succeeded'
+                  ]
+                }
+              ]
+              userProperties: []
+              typeProperties: {
+                variableName: 'exportName'
+                value: {
+                  value: '@if(greater(length(replace(tolower(concat(pipeline().parameters.StorageAccountName,\'oamortized\', last(split(item().scope, \'/\')))), \'-\', \'\')), 64), substring(replace(tolower(concat(pipeline().parameters.StorageAccountName,\'oamortized\', last(split(item().scope, \'/\')))), \'-\', \'\'), 0, 63), replace(tolower(concat(pipeline().parameters.StorageAccountName,\'oamortized\', last(split(item().scope, \'/\')))), \'-\', \'\'))'
```

Comment:
Can we move the metric to the end? Maybe `FinOpsHub-${hubName}-${metric}`? We can remove the prefix if it's too long. My goal is just to group them together in the list. We probably don't care about the scope or storage account names in this.

Reply:
I do need to differentiate between daily (open) and monthly (closed) exports, so how about:
${hubName}-daily-${metric} 
${hubName}-monthly-${metric}